requirements.txt

```diff
@@ -5,4 +5,8 @@ networkx>=2.4
 pytest
 pandas>=1.0.0
 polars>=0.16.6
-pre-commit
\ No newline at end of file
+pre-commit
+kloppy~=3.11.0
+pytz~=2023.3
+python-dateutil~=2.8.2
+setuptools~=67.8.0
```

Comment:
What is the intention behind having a setuptools entry in requirements.txt? I think you can’t install Setuptools from the requirements.txt file anyway because you need to have Setuptools installed in order to install anything (with Pip) in the first place.

Reply:
@MKlaasman can you update this one?